flycheck-rust.el

```diff
@@ -134,7 +134,7 @@ description of the conventional Cargo project layout."
             ;; that one
             (seq-find (lambda (target)
                         (let-alist target (string= file-name .src_path)))
-                      targets)
+                      (-flatten-n 1 targets))
```

Comment:
I think we should rename `targets` to `packages` in the `-when-let*` above, since that's what they are.  We could then add a `(targets (-flatten-n 1) packages)` binding to that `-when-let*`, to avoid duplication.